pkgs/development/compilers/graalvm/community-edition/buildGraalvm.nix

```diff
@@ -118,13 +120,23 @@ let
       xorg.libXi
       xorg.libXrender
       xorg.libXtst
-    ];
+    ] ++ (lib.optionals stdenv.hostPlatform.isDarwin [
+        apple-sdk_11
+        (darwinMinVersionHook "11.0")
+      ]);
 
     postInstall =
       let
         cLibsAsFlags = (map (l: "--add-flags '-H:CLibraryPath=${l}/lib'") cLibs);
         preservedNixVariables = [
+          "-EDEVELOPER_DIR"
+          "-EDEVELOPER_DIR_FOR_BUILD"
+          "-EDEVELOPER_DIR_FOR_TARGET"
           "-ELOCALE_ARCHIVE"
```

Comment:
I am almost sure that this is Linux only, so could you move this to `lib.optionals stdenv.hostPlatform.isLinux`?